seeding.py

```diff
@@ -114,8 +114,21 @@ def get_db():
     return MongoClient(conf.MONGO_URI, uuidRepresentation="standard")[conf.MONGO_DBNAME]
 
 
+def seed_jobs(db, seed_path):
+    with open(seed_path, "r") as f:
+        jobs = json.load(f)
+
+    for job in jobs:
```

Comment:
if we add an static uuid in the seeding we can check if it exists in the db already and avoid inserting the jobs multiple times